appveyor.yml

```diff
@@ -73,4 +75,4 @@ test_script:
     # Generate recipes...
     - "%CMD_IN_ENV% extrude_recipes requirements.yml"
     # Build, and, if necessary, upload.
-    - if exist recipes %CMD_IN_ENV% conda build-all recipes --inspect-channels %DESTINATION_CONDA_CHANNEL% conda-forge openastronomy astropy --matrix-condition="python %PYTHON_BUILD_RESTRICTIONS%"  %UPLOAD%
+    - if exist recipes %CMD_IN_ENV% conda build-all recipes --inspect-channels %DESTINATION_CONDA_CHANNEL% conda-forge openastronomy astropy --matrix-conditions "python %PYTHON_BUILD_RESTRICTIONS%" "%NUMPY_BUILD_RESTRICTIONS%" %UPLOAD%
```

Comment:
don't use `"` around ``NUMPY_BUILD_RESTRICTIONS``, as it doesn't work when it's set to `""` above.